docs/get-started.mdx

```diff
@@ -0,0 +1,231 @@
+which we can't maintain and are often missing features. By installing from a method listed here, you'll get the complete
+Netdata experience.
+
+## Install Netdata
+
+Choose your installation method below.
+
+The best way to install Netdata is to use the **[one-line kickstart script on Linux](#install-netdata-on-linux)** or
+**[run with Docker](#run-netdata-with-docker)**. Many Linux distributions ship with third-party packages of Netdata,
+which we can't maintain and are often missing features. By installing from a method listed here, you'll get the complete
+Netdata experience.
+
+<Install>
+  <InstallBox
+    to="#install-netdata-on-linux"
+    os="General Linux with one-line installer"
+    svg="linux" />
+  <InstallBox
+    to="#run-netdata-with-docker"
+    os="Run with Docker" 
+    svg="docker" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/packages"
+    os="Linux with .deb/.rpm packages" 
+    svg="linux" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/macos"
+    os="Install on macOS" 
+    svg="macos" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/kubernetes"
+    os="Deploy on Kubernetes" 
+    svg="kubernetes" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/manual"
+    os="Linux from Git" 
+    svg="linux" />
+</Install>
+
+<details>
+  <summary><strong>Need a different installation method, or use a different operating system? See all our options.</strong></summary>
+  <Install>
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kickstart"
+      os="Linux with one-line installer/kickstart script" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/packages"
+      os="Linux with .deb/.rpm packages" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kickstart-64"
+      os="Linux with static 64-bit binary" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/manual" 
+      os="Linux from Git" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/source"
+      os="Linux from source" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/offline" 
+      os="Linux for offline nodes" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/docker"
+      os="Docker" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kubernetes"
+      os="Kubernetes" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/macos"
+      os="macOS" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/cloud-providers"
+      os="Cloud providers (GCP, AWS, Azure)" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/freebsd"
+      os="FreeBSD"
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/pfsense"
+      os="pfSense" 
+      community />
```

Comment:
In the drop down for installation boxes, we need a line that defines the "community-supported". 

Also in the installation docs we need to link to the original packages and maintainer name so they know where to go in-case it doesn't work. I suggest we remove all these and we tackle them properly in a different repo.

Let's do it right.